sonar-project.properties

```diff
@@ -0,0 +1,6 @@
+sonar.projectKey=LedgerHQ_ledger-live_AYoXRskslck2EnGRKg2f
+sonar.sources=apps/ledger-live-desktop/src/
+sonar.exclusions=**/*.java,**/node_modules/,**/.turbo,**/.pnpm-*,**/temp,**/.eslintcache,**/tsconfig.tsbuildinfo,**/*.png,**/*.env
+sonar.coverage.exclusions=**/*.java
+sonar.tests=apps/ledger-live-desktop/tests/
+sonar.javascript.lcov.reportPaths=apps/ledger-live-desktop/coverage/lcov.info
```

Comment:
If you want to see if LLC coverage can be reported to, here's LLC report path: libs/ledger-live-common/coverage/lcov.info